client/src/connection/itc/script.ts

```diff
@@ -75,13 +77,19 @@ class SASRunner{
 
   [void]Run([string]$code) {
     try{
+        $this.objSAS.LanguageService.Reset()
+        $this.objSAS.LanguageService.Async = $true
         $this.objSAS.LanguageService.Submit($code)
     }catch{
       Write-Error $_.ScriptStackTrace
       throw "Run error"
     }
   }
 
+  [void]SubmitComplete() {
```

Comment:
nit: looks like not used?

Reply:
😬 You're right. Updated in https://github.com/sassoftware/vscode-sas-extension/pull/592/commits/a69deff13525927c73f45d3447b8ecd95ac2600c